src/NumField/Selmer.jl

```diff
@@ -281,16 +281,16 @@ function Hecke.is_power(a::FacElem{QQFieldElem, QQField}, p::Int)
   return true
 end
 
-function Hecke.is_power_with_root(a::FacElem{QQFieldElem, QQField}, p::Int)
+function is_power_with_root(a::FacElem{QQFieldElem, QQField}, p::Int)
```

Comment:
```suggestion
function is_power(a::FacElem{QQFieldElem, QQField}, p::Int)
```
This should actually be renamed as well, similar to https://github.com/Nemocas/AbstractAlgebra.jl/pull/1529